CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set( CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS TRUE )
 
 # Define here the needed parameters
 set (OPENRAVE_VERSION_MAJOR 0)
-set (OPENRAVE_VERSION_MINOR 58)
+set (OPENRAVE_VERSION_MINOR 59)

```

Comment:
@Puttichai it's not clear why you are changing minor version...

Reply:
Following our discussion, I reverted the change of the minor version and increased the patch version instead.